README.md

```diff
@@ -92,12 +92,9 @@ make
 ```
 
 ## Usage
-
-* [One click KubeEdge Installer to install both Cloud and Edge nodes](./docs/setup/installer_setup.md)
-* [Run KubeEdge from release package](./docs/getting-started/release_package.md)
-* [Run KubeEdge from source](./docs/setup/setup.md)
-* [Deploy Application](./docs/setup/setup.md#deploy-application-on-cloud-side)
-* [Run Tests](./docs/setup/setup.md#run-tests)
+* [Install KubeEdge](./docs/setup/kubeedge_install.md)
+* [Configure KubeEdge](./docs/setup/kubeedge_configure.md)
```

Comment:
@fisherxu Hope you are doing well. (I might be wrong) We have tried to present the KubeEdge in different stages to the user for ease of understanding. Also, to avoid adding everything in one file lengthy documentation. To run KubeEdge the user has to install KubeEdge (involves) 
- Installing: the pre-req, installation (can be compiling edgecore, cloudcore)
- Configure: Creating the configuration files and setting necessary values.
- Run: Mainly running the binary (cloudcore/ edgecore) or setting them as systemd services.
- PreCheck: Making sure everything is correctly setup by running `kubectl get nodes` etc.

I believe, if we keep this logically it helps users to follow step by step procedures.

Yours Sincerely,
Vijay

Reply:
> Installing: the pre-req, installation (can be compiling edgecore, cloudcore)

This will only in sourcecode installation, when we use keadm to install, it's no need to install and run